docs/src/lib/indexing.md

```diff
@@ -212,8 +212,9 @@ performance on par with integer indexing.
 The elements of a `GroupedDataFrame` are [`SubDataFrame`](@ref)s of its parent.
 
 * `gd[i::Integer]` -> Get the `i`th group.
-* `gd[a::AbstractArray{<:Integer}]` -> Get a reduced `GroupedDataFrame` containing only the groups
-  with indices in `a`.
+* `gd[a::AbstractArray{<:Union{Integer, Boolean}}]` -> Use array-style indexing
```

Comment:
`Boolean` type does not exist. If you meant `Bool` it is a subtype of `Integer` so no need to mention it. However, we discussed of adding `Not` support in this PR.